README.adoc

```diff
@@ -97,6 +110,9 @@ public class BindingInitializer {
         .setUsername("myuser") // <2>
         .setPassword("mypassword"); // <3>
         .setSecure(true) // <4>
+        .setConnectTimeout(10000) // <5>
```

Comment:
Looks better i was thinking of that in the beginning but decided to stick to the same data type the library uses. But im ok with it. This way it is better readable and think noboady will try to set the timeout higher than integer max value milliseconds ;) i will look at your other reviews tomorrow and do the cleanups

Reply:
Regarding timeout  again please consider the following for all timeout values:
On the one hand it is more transparent by just pass the values through with the same data type like they are used in the rabbitmq library without any conversion.

Also Using TimeUnits could cause wrong communication to the user of the library as for some timeouts only seconds are allowed. 

Additional to be correct checking of values is needed so there is no number overflow: conversion from long to int. Throwing IllegalArgumentException in this case could be a solution.

Using a set of mixed api is another solution for some values ints and for some TimeUnit but this looks unclean to me.

On the other hand it offers better readability because you clearly can see that 10 means seconds like in your example.
I think there is no real winner here, awaiting your decision.